packages/oui-tooltip/src/tooltip.html

```diff
@@ -0,0 +1,13 @@
+<div class="oui-tooltip oui-message" role="tooltip">
+  <span aria-hidden="true"
+    ng-class="{
+      'oui-tip-top':                  (!$ctrl.flipped && $ctrl.placement === 'top') || ($ctrl.flipped && $ctrl.placement === 'bottom'),
```

Comment:
Is this part can be simplified in a method?
It will be simplified when if it remains only top|bottom|left|right , but still...
If I read the guidelines:

> Default opening direction is "top center"
The content opens inside the viewport in the direction where there is the most available space.

So there is no place for a "placement" attribute. (I think you should aven don't use end/start suffixes in the case of the tooltips).

Another point: one-time binding maybe?

Reply:
I'll look to try a simpler way to do that. But now with the management of the arrow by Popper, it'll be.

If one-time, i can't change it when flipped.